client/components/plans/plan-icon/index.jsx

```diff
@@ -0,0 +1,497 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import {
+	PLAN_FREE,
+	PLAN_PREMIUM,
+	PLAN_BUSINESS,
+	PLAN_JETPACK_FREE,
+	PLAN_JETPACK_BUSINESS,
+	PLAN_JETPACK_BUSINESS_MONTHLY,
+	PLAN_JETPACK_PREMIUM,
+	PLAN_JETPACK_PREMIUM_MONTHLY,
+	PLAN_PERSONAL
+} from 'lib/plans/constants';
+
+export default class PlanIcon extends Component {
+	render() {
+		const { plan } = this.props;
+
+		return (
+			<div className="plan-icon">
```

Comment:
If we do add a default width/height, note that it will increase the css specificity needed to override the base style.

Our current gridicons take a size prop (which are set as width and height attrs on the svg), but in a number of cases we override this default sizing with CSS, which can lead to browser bugs ( in IE11) when we incorrectly try to reset this. 

For our current use case lets just use classes for now, since these illustrations will be larger and will probably have a number of breakpoints on pages.